_data/committee/2021-22.yaml

```diff
@@ -115,6 +138,18 @@ people:
       Hello, I'm Ryan and I'm going into my 2nd year studying Computer Science. I will be responsible for all of your sports and e-Sports events this year -- 
       please let me know what you want to see! When I'm not sleeping I'm either speedrunning, participating in hackathons or working on programming projects.
 
+  - name: Aarogya Yadav
+    pronouns: He/Him
+    discordtag: Arcane34#4178
+    role: Sports Secretary
+    picture: /assets/committee/2021-22/mini/aarogya.jpg
+    start-date: 29/1/2022
```

Comment:
Aarogya, Tori and Matty started on 15/02/2022

Reply:
Fixed